lib.nix

```diff
@@ -77,7 +77,7 @@ let
           in
 
           let nickel_expr | params.nix.NickelExpression =
-            import "${sources}/${nickelFile}"
+            import "${nickelFile}"
```

Comment:
fixing ` > error: import of /nix/store/p6qc3j8vgfj1c918m04mg4hgsln4lwq0-c-hello-world//nix/store/i4bm1rmvjn37iz4fmn7bshwb3jgfg81x-hello.ncl failed: No such file or directory (os error 2)`